usps/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/corbaltcode/usps/usps
```

Comment:
Would it make sense to have a single module for all our USPS stuff — `github.com/corbaltcode/usps`?

If that were the case, we could have separate packages for the [City State Product](https://postalpro.usps.com/address-quality/city-state-product), [ZIP+4 Product](https://postalpro.usps.com/address-quality-solutions/zip-4-product), and EPF (as you initially did in 0613714e42a19249268cdd2e30ba04472ba4319c).

Reply:
I think it'd make sense to get this sorted before the initial merge. `zip4` has a dependency on `citystate` because the ZIP+4 Product contains a City State file. Are there other dependencies or ways the functionality is mixed?

Reply:
I believe that is the only dependency

Reply:
I've separated citystate.go and ziptools.go back out into their own packages, and confirmed that the ziptools repository still works with this change; PTAL!